### PR TITLE
Encode salt signature as base64 

### PR DESCRIPTION
**Pull request by mchugh19**
Avoids passing wacky characters through salt's message bus. 

Addresses issue #10

### CODE REVIEW

File: _utils/vault.py [hunk 1]
```diff
@@ -26,7 +27,7 @@ def _get_token_and_url_from_master():
   if __opts__.get('__role', 'minion') == 'minion':
     private_key = '{0}/minion.pem'.format(pki_dir)
     log.debug('Running on minion, signing token request with key {0}'.format(private_key))
-    signature = salt.crypt.sign_message(private_key, minion_id)
+    signature = base64.b64encode(salt.crypt.sign_message(private_key, minion_id))
```
Comment (carlpett) [inline _utils/vault.py:30]: This should be on the corresponding line for encrypting with the master key as well

**Comment by carlpett** [comment]
One minor thing, but otherwise very nice find!

**Comment by mchugh19** [comment]
Pushed update to cover master base64 encoding. Thanks for getting this going!

**Comment by carlpett** [comment]
Nice, thanks! 
Also, FYI, the merge into mainline is going through review now, so hopefully it will be bundled with the next release of Salt. Thanks for the reminder in the other thread :)

**Pull request merged**